package/gluon-core/luasrc/lib/gluon/upgrade/115-swconfig

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/lua
+
+local platform = require 'gluon.platform'
+local uci = require('simple-uci').cursor()
+
+local function get_keys(t)
+	local keys = {}
+	local n = 0
+
+	for k,_ in pairs(t) do
+		n = n+1
+		keys[n] = k
```

Comment:
```suggestion
		table.insert(keys, k)
```
While using an explicit counter variable is slightly more efficient, this shouldn't matter here, and I consider `table.insert()` more idiomatic Lua.

Reply:
outdated